.circleci/workflows.yml

```diff
@@ -498,8 +498,9 @@ commands:
     steps:
       - run: pwd
       - run:
-          name: print global yarn cache path
-          command: echo $(yarn global bin)
+          name: print global yarn cache path and size
+          command: |
+            du -sh $(yarn global bin)
```

Comment:
we talked about the cache size, but I couldn't find that size anywhere, so this should print the size for tracking changes if we wanted. See this here: https://app.circleci.com/pipelines/github/cypress-io/cypress/69319/workflows/3aa29fa3-b62d-48c0-9a8a-ca17cec1ea79/jobs/2849177